### DISCUSSION TITLE
Refresh API Per Second

### DISCUSSION THREAD

**Discussion by takai-sama** [grafana/grafana-json-datasource#204] [Q&A] [answered]
can this plugin get APIs per second? and refresh data per second?
I have an API that changes every second, but when using this plugin, the data doesn't refresh per second even though I have set the dashboard refresh per second (1s). any ideas?

**Comment by marcusolsson** [comment, marked as answer]
The data source caches data by default to avoid unintentionally hit rate limits. If you want to disable the cache, set the cache duration to 0s.
